modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingWorkflowView.js

```diff
@@ -3,94 +3,145 @@
-							{Liferay.Language.get('view-usages')}
-						</a>
-					</ClayTable.Cell>
-				</ClayTable.Row>
-
-				<ClayTable.Row>
-					<ClayTable.Cell className="publications-key-td table-cell-expand-small">
-						{Liferay.Language.get('comments')}
-					</ClayTable.Cell>
-
-					<ClayTable.Cell className="table-cell-expand">
-						{workflowData.comments}
-					</ClayTable.Cell>
-				</ClayTable.Row>
-			</ClayTable.Body>
-		</ClayTable>
+		<div>
+			<ClayTable
+				borderless
+				className="publications-render-table table table-autofit table-nowrap"
+			>
+				<ClayTable.Head />
+
+				<ClayTable.Body>
+					<ClayTable.Row>
+						<ClayTable.Cell className="publications-key-td table-cell-expand-small">
+							{Liferay.Language.get('status')}
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							<WorkflowStatusLabel
+								workflowStatus={workflowData.status}
+							/>
+						</ClayTable.Cell>
+					</ClayTable.Row>
+
+					<ClayTable.Row>
+						<ClayTable.Cell className="publications-key-td table-cell-expand-small">
+							{Liferay.Language.get('assigned-to')}
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							{workflowData.assignedTo}
+						</ClayTable.Cell>
+					</ClayTable.Row>
+
+					<ClayTable.Row>
+						<ClayTable.Cell className="publications-key-td table-cell-expand-small">
+							{Liferay.Language.get('task-name')}
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							{workflowData.taskName}
+						</ClayTable.Cell>
+					</ClayTable.Row>
+
+					<ClayTable.Row>
+						<ClayTable.Cell className="publications-key-td table-cell-expand-small">
+							{Liferay.Language.get('create-date')}
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							{workflowData.createDate}
+						</ClayTable.Cell>
+					</ClayTable.Row>
+
+					<ClayTable.Row>
+						<ClayTable.Cell className="publications-key-td table-cell-expand-small">
+							{Liferay.Language.get('due-date')}
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							{workflowData.dueDate}
+						</ClayTable.Cell>
+					</ClayTable.Row>
+
+					<ClayTable.Row>
+						<ClayTable.Cell className="publications-key-td table-cell-expand-small">
+							{Liferay.Language.get('usages')}
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							<a href={Liferay.Util.escape(workflowData.usages)}>
+								{Liferay.Language.get('view-usages')}
+							</a>
+						</ClayTable.Cell>
+					</ClayTable.Row>
+
+					<ClayTable.Row>
+						<ClayTable.Cell className="publications-key-td table-cell-expand-small">
+							{Liferay.Language.get('comments')}
+						</ClayTable.Cell>
+
+						<ClayTable.Cell className="table-cell-expand">
+							{workflowData.comments}
+						</ClayTable.Cell>
+					</ClayTable.Row>
+				</ClayTable.Body>
+			</ClayTable>
+
+			<ClayPanel
```

Comment:
Is it possible to have nested tables? I think the spacing will look nicer if we have the ClayPanel in the ClayTable